shared/qt/idian/widgets/Row.hpp

```diff
@@ -0,0 +1,211 @@
+/******************************************************************************
+    Copyright (C) 2023 by Dennis Sädtler <dennis@obsproject.com>
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+******************************************************************************/
+
+#pragma once
+
+#include <QWidget>
+#include <QFrame>
+#include <QLabel>
+#include <QLayout>
+#include <QScrollArea>
+#include <QMouseEvent>
+#include <QCheckBox>
+
+#include "../components/ToggleSwitch.hpp"
+#include "../components/ComboBox.hpp"
+#include "../components/SpinBox.hpp"
+#include "../components/DoubleSpinBox.hpp"
+#include "../Utils.hpp"
+#include "PropertiesList.hpp"
```

Comment:
Includes still need to be changed to not be relative (in many files, not just this one). Marking this for myself.

Reply:
Changed this. (again, [previous version retained](https://github.com/gxalpha/obs-studio/tree/idian-rebase-backup-before-include-changes))